test/metabase/task/sync_databases_test.clj

```diff
@@ -38,6 +41,26 @@
                                             (update    :key  replace-trailing-id-with-<id>)
                                             (update-in [:data "db-id"] replace-trailing-id-with-<id>))))))))))
 
+(defn all-db-sync-triggers-name
+  "Returns the name of trigger for DB.
+  This is all trigger names that a DB SHOULD have."
```

Comment:
```suggestion
  These are all the trigger names that a database SHOULD have."
```

The singular compound non sounded weird to me. The uppercase DB was fighting with highlighting the modal verb.